scripts/jenkins/cloud/manual/input.yml.example

```diff
@@ -206,6 +206,10 @@ ses_enabled: true
 # if ses_enabled is set to true.
 ses_rgw_enabled: false
 
+#============= OpenID Connect =============#
+# Enable OpenID Connect feature in Keystone and configure it with a bogus IdP
+# for validation purposes (i.e. make sure nothing breaks by default).
+openid_connect_enabled: true
```

Comment:
Technically this should also be added to one of the files under the ../ansible/group_vars/all/... files.

Reply:
Yes, it's in all.yml as well.

Reply:
Okeydokey - missed the line above ;-)